app/models/policies/international_relocation_payments.rb

```diff
@@ -1,4 +1,12 @@
 module Policies
   module InternationalRelocationPayments
+    include BasePolicy
+    extend self
+
+    # NOTE RL: currently IRP only has a single reply to address, so notify
+    # doesn't show the address id
+    def notify_reply_to_id
+      nil
+    end
```

Comment:
if we're migrating to use to the claim notify account i would have expected this to be populated?

Reply:
We can migrate the templates separately.